README.md

```diff
@@ -61,6 +61,18 @@ RUN c_rehash /etc/ssl/certs
 RUN zypper --gpg-auto-import-keys ref -s
 RUN zypper -n in vim
 ```
+
+All recommended package modules are enabled by default. It is possible to enable
+additionally non-recommended modules via the environment variable
+`ADDITIONAL_MODULES`:
+```
+FROM registry.suse.com/suse/sle15:latest
+
+ENV ADDITIONAL_MODULES sle-module-desktop-applications,sle-module-development-tools
+
+RUN zypper --gpg-auto-import-keys ref -s
+RUN zypper -n in gvim
+```
```

Comment:
The documentation should be mention the module names are the `identifier`. Also we should find a way to print out a list of available modules to the user. See my proposal at the bottom of the PR.

Reply:
Yep, for now I added the mentioned `identifier`.